service-loadbalancer/service_loadbalancer.go

```diff
@@ -149,6 +149,10 @@ type service struct {
 
 	// Algorithm
 	Algorithm string
+
+	// Indicates if the service must use sticky sessions
+	// http://cbonte.github.io/haproxy-dconv/configuration-1.5.html#stick-table
+	SessionAffinity bool
```

Comment:
please update the README about how to use/set this, also document the other annotations we've recently added if they're not already there